docs/dmytro-shevchenko1/GamesProject/styles/style.css

```diff
@@ -0,0 +1,13 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
```

Comment:
I think you should make a `body` have the min height of the screen and flex column container. And then make `main` flex-growing to get all free place. 
Now you have a lot of empty spaces between the main content and the footer
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/446398e1-5c1a-435a-a4c3-6f86982e2baf)